test/selenium/lib/view/log_in/accessors.js

```diff
@@ -0,0 +1,24 @@
+var By = require('selenium-webdriver').By;
+var Accessors = require('../accessors');
+
+
+function LoginAccessors() {
+  Accessors.apply(this, arguments);
+}
+
+LoginAccessors.prototype = Object.assign({
+
+  get passwordField() {
+    return this.waitForElement(By.id('signup-pwd1'));
+  },
```

Comment:
I know this and the other two accessors are defined for [here](https://github.com/fxbox/foxbox/blob/master/test/selenium/lib/view/sign_up/accessors.js#L6) but that way, they are not accessible from the app, need to do this.